src/main/java/io/jenkins/plugins/ml/KernelInterpreter.java

```diff
@@ -0,0 +1,12 @@
+package io.jenkins.plugins.ml;
+
+public interface KernelInterpreter {
+
+    /**
+     * Interface for Kernel Interpreter (Eg: IPython ,R and Julia).
+     */
```

Comment:
I think this is the javadoc for the interface? Not for this method? If so, this comment should be moved to the top of the `public interface KernelInterpreter`.

Reply:
Yes its for the interface. will shift up.